objc/core/A0LockReact.m

```diff
@@ -44,6 +49,19 @@ - (void)configureLockWithClientId:(NSString *)clientId domain:(NSString *)domain
     _lock = [A0Lock newLockWithClientId:clientId domain:domain];
 }
 
+- (void)hideWithCallback:(A0LockCallback)callback {
+    if (self.shown) {
+        if (!self.lock) {
+            callback(@[@"Please configure Lock before using it"]);
+            return;
+        }
+        UIViewController *controller = [[[[UIApplication sharedApplication] windows] firstObject] rootViewController];
+        [controller dismissViewControllerAnimated:YES completion:nil];
+        self.shown = NO;
+    }
+    callback(@[]);
```

Comment:
I think we should call the callback inside the `completion` block of `dismissViewControllerAnimated:completion:`